scripts/postinstall.sh

```diff
@@ -1,15 +1,4 @@
 #!/bin/bash
 
-# Add commonjs entry points to pure ESM modules. Needed by running node tests.
-npx babel node_modules/@mapbox/tiny-sdf/index.js > node_modules/@mapbox/tiny-sdf/index.cjs
```

Comment:
I guess CJS applications using deck still need to do this?

Reply:
Well that's not really within our control..